src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/VisibleMemberTable.java

```diff
@@ -1006,6 +1008,8 @@ public ImplementedMethods(ExecutableElement method) {
              * from the list.
              */
             for (TypeMirror interfaceType : intfacs) {
+                // TODO: this method also finds static methods which are pseudo-inherited;
+                //  this needs to be fixed
```

Comment:
yeah, there's (at least) two abstractions here: visible members (note the name of the class!) and the implemented members.   Note that the `@see`/`@link` syntax doesn't distinguish static or not, it just wants "visible".